### PR TITLE
temp fix for serialport on macos 11

### PR DESCRIPTION
**Pull request by spookyvision**

### CODE REVIEW

File: xtask/src/tasks.rs [hunk 1]
```diff
@@ -198,7 +198,7 @@ pub fn serial_term() -> color_eyre::Result<()> {
         }
     };
 
-    let mut port = serialport::open(&dongle.port_name)?;
+    let mut port = serialport::new(&dongle.port_name, 9600).open()?;
```
Comment (jamesmunns) [inline xtask/src/tasks.rs:201]: 9600 is pretty low, would you mind trying this with `115200`, which is a pretty standard speed as well?

**Review by jamesmunns — APPROVED** [review]
Overall looks good to me! Could you please open an issue to revert this fix, and maybe mention it in the comments where you introduce your fork of serialport?

We might also want to move this fork to the Ferrous space.

File: xtask/src/tasks.rs (hunk 1 above)
Comment (spookyvision) [reply xtask/src/tasks.rs:201]: 9600 was the default speed for `serialport::open` (I checked :D )
115200 works for `dongle-flash` and `serial-term` (actually now I get regular packets of `received 45 bytes (CRC=Ok(0xc0ba), LQI=15)`; previously I only saw the double `now listening on channel 20` from troubleshooting)

**Comment by spookyvision** [comment]
> Overall looks good to me! Could you please open an issue to revert this fix, and maybe mention it in the comments where you introduce your fork of serialport?

[issue created](https://github.com/ferrous-systems/embedded-trainings-2020/issues/148) - I don't understand the "mention in the comments..." part

> We might also want to move this fork to the Ferrous space.
it's on git_lab_ - do we have anything there?

File: xtask/src/tasks.rs (hunk 1 above)
Comment (jamesmunns) [reply xtask/src/tasks.rs]: Let's switch it over, we can beta test in the training tomorrow, and hotpatch it back quickly if we need to :)

Comment (spookyvision) [reply xtask/src/tasks.rs]: already did!

File: xtask/Cargo.toml [hunk 2]
```diff
@@ -12,6 +12,7 @@ hidapi = "1.2.2"
 ihex = "1.1.2"
 pids = { path = "../common/pids" }
 rusb = "0.5.5"
-serialport = "3.3.0"
+#serialport = "3.3.0"
```
Comment (jamesmunns) [inline xtask/Cargo.toml:15]:
```suggestion
# Note: we are using a fork due compatibility problems.
# See https://github.com/ferrous-systems/embedded-trainings-2020/issues/148 for more details.
#serialport = "3.3.0"
```

**Comment by jamesmunns** [comment]
> it's on git lab - do we have anything there?

I'd say:

* Do a clone
* Make a new github repo
* Push the clone to the new repo

**Pull request merged**